### DISCUSSION TITLE
why you close my issues ,are you fix?

### DISCUSSION THREAD

**Discussion by newyorkthink** [mmtrt/WINE_AppImage#55] [General]

**Comment by mmtrt** [comment]
well most of the issues were redundant and for other issues were closed as won't fix.

**Comment by kenderipa** [comment]
> well most of the issues were redundant and for other issues were closed as won't fix.

So did you actually merge nVidia libraries into AppImages?

**Reply by mmtrt** [reply]
This won't be needed once I change stable,devel,staging to wine wow64 appimages.

**Reply by kenderipa** [reply]
Oh, all right then.

**Reply by mmtrt** [reply]
change was done as of today should be working now.

**Reply by kenderipa** [reply]
If doesn't work, if I'm trying to start a game from Heroic Games Laucher with continuous-stagin AppImage.
